style.css

```diff
@@ -0,0 +1,229 @@
+body {
+  background-color: #fff;
+  font-family: Helvetica, sans-serif;
+  margin:0;
+}
+
+h2 {
+  font-size: 30px;
+  line-height: 30px;
+  text-align: center;
+}
+
+h3 {
+  font-size: 30px;
+  text-align: center;
+}
+
+p {
+text-align: center;
+}
+
+.footer {
+  background-color: black;
+  color: white;
+  margin: 0;
+  padding-top: 5px;
+  padding-bottom: 5px;
+}
+
+.container {
+  width: 100%;
+  max-width: 1100px;
+  margin: 0 auto;
+}
+
+/*** HEADER ***/
+
+header {
+  background: black;
+  height: 80px;
+  display: flex;
+  align-items: center;
+  padding: 0 16px;
+}
+
+.hamburger {
+  display: flex;
+  width: 50px;
+  height: 50px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-right: 24px;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background: #fff;
+  border-radius: 5px;
+}
+
+header img {
+  width: 50px;
+  padding: 5px;
+}
+
+nav {
+  display: none;
+}
+
+nav a {
+  color: #fff;
+  text-decoration: none;
+  margin-left: 16px;
+}
+
+
+/*** WELCOME CONTAINER ***/
+
+.welcome-container .container { 
+  padding: 100px 16px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+}
+
+.welcome-container img {
+  width: 100%;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.welcome-container div {
+  text-align: center;
+}
+
+
+/*** KITCHEN CONTAINER ***/
+
+.kitchen-container {
+  background: #fff;
+  color: #000;
+  text-align: center;
+  padding: 0px;
+}
+
+.kitchen {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.kitchen a {
+  border: 2px dotted #24292d;
+  border-radius: 16px;
+  width: 100%;
+  text-decoration: none;
+  color: #000;
+  padding: 16px;
+  margin-bottom: 32px;
+}
+
+.kitchen a div {
+  border: 2px  #d8eee6;
+  border-radius: 16px;
+}
+
+.kitchen a img {
+  width: 80%;
+}
+
+
+/*** INTIRIOR CONTAINER ***/
+
+.interior-container {
+  background: #fff;
+  color: #000;
+  text-align: center;
+  align-items: center;
+  padding: 24px;
+}
+
+.interior {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.interior a {
+  width: 100%;
+  height: 100%;
+  text-decoration: none;
+  color: #000;
+  padding: 16px;
+  margin-bottom: 32px;
+}
+
+.interior a img {
+  width: 50%;
+  height:50%;
+}
+
+.interior a div {
+  border: 2px sold #d8eee6;
+  border-radius: 16px;
+}
+
+.interior-square-images {
```

Comment:
Consider changing the class name as these are not square ;) Looking really good!